beacon/META.yml

```diff
@@ -1,4 +1,7 @@
 links:
+    - label: webperf
+      results:
+        - test: *
```

Comment:
Looks like all the existing metadata ends up quoting `*` like `test: '*'`. I'm not sure if that avoids some YAML pitfall, but I"m not sure it doesn't, so let's be consistent.